coss/opensource_clubs/models.py

```diff
@@ -7,13 +7,21 @@
 from wagtail.wagtailcore.fields import RichTextField
 from wagtail.wagtailcore.models import Orderable, Page
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
-
+from wagtail.wagtailsnippets.edit_handlers import SnippetChooserPanel
 
 class CategoryLandingPage(Page):
     description = RichTextField(blank=True)
+    discourse = models.ForeignKey(
+        'discourse.DiscourseCategoryLatest',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
```

Comment:
Right, but it seems to me like if a user deletes a Discourse snippet, they'd just want it removed from the landing page, rather than having the landing page be deleted too.